src/test/ui/rfc-2397/feature-gate-do_not_recommend.rs

```diff
@@ -0,0 +1,10 @@
+// check-pass
```

Comment:
Would it make more sense to rename the directory from `rfc-2397` (which doesn't really tell what it's about) to `do-not-recommend`?

Reply:
Renamed to rfc-2397-do-not-recommend